### ISSUE TITLE
ONNX input

### ISSUE THREAD

**Issue by omidb** [AILab-CVC/YOLO-World#27]
Currently, while exporting the YOLOWorldDetector model to onnx format, the text input for the model is locked for all possible input images at inference time. 
Can onnx model weights where the model accepts both image and text as input be shared? 
Otherwise can the pytorch to onnx conversion script be modified s.t. it is also possible to provide text (tokenized input ids and attention mask) as input to onnx model at inference time.

**Comment by Freedomeri** [comment]
The same question for me >_<...
Hope for solutions!

**Comment by wondervictor** [comment]
Hi @Freedomeri @omidb, I'll update with an ONNX demo.

**Comment by MarxMelencio** [comment]
@wondervictor  - Cool! Also, is the ONNX demo that accepts text and image inputs already in the repo somewhere? This would be really helpful for our use case! Thanks.
